openpype/hosts/fusion/scripts/fusion_switch_shot.py

```diff
@@ -46,12 +47,8 @@ def _format_version_folder(folder):
 
 def _get_work_folder(session):
     """Convenience function to get the work folder path of the current asset"""
-
     # Get new filename, create path based on asset and work template
-    template_work = self._project["config"]["template"]["work"]
-    work_path = pipeline._format_work_template(template_work, session)
-
-    return os.path.normpath(work_path)
+    return get_workdir_from_session(session)
```

Comment:
Should we remove the `_get_work_folder` function and refactor other code to just use `get_workdir_from_session` directly?
Especially since it's now just a single function call. I'd argue the same for the removal of [`_get_context_directory` in `switch_ui.py`](https://github.com/pypeclub/OpenPype/pull/2839/files#diff-202acaf33ac1ec4e4421ad5a56783a07d1dc51ff2da3ee3d62aaf8cca8c399f1R161)

Reply:
Done. I actually wasn't sure if the script is even used?

Reply:
Yes, it's old - probably unused in OpenPype - and it's been a while since we have used it too. We could consider it code debt for a future separate PR and discuss a more thorough cleanup.